src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1971,6 +1971,32 @@ public void cannotCreateRequestForClaimedReturnedItem(String requestType) {
     )));
   }
 
+  @Parameters({
+    "",
+    "Recall",
+    "Hold",
+    "Page"
+  })
+  @Test
+  public void cannotCreateRequestForWithdrawnItem(String requestType) {
```

Comment:
Could these tests be unit like tests rather than API tests?

Reply:
There is already a unit test `src/test/java/api/requests/RequestTypeItemStatusWhiteListTests.java#cannotCreateRequestWhenItemStatusWithdrawn`.

Should I remove this API test then?

Reply:
I think we only really need one API test to demonstrate that the whitelist is being applied. The rest can be demonstrated by unit tests.

I'd love to leave them in place to demonstrate the overall behaviour of the system, however there is a trade off to that, as the API tests are expensive to execute.

What do you think?

Reply:
Removed. We can create a spike to identify how API tests execution time can be improved.